src/action/auth.js

```diff
@@ -0,0 +1,38 @@
+import superagent from 'superagent'
+import * as util from '../lib/util.js'
+window.util = util
+export const tokenSet = (token) => ({
+  type: 'TOKEN_SET',
+  payload: token,
+})
+
+export const tokenRemove = () => ({
+  type: 'TOKEN_REMOVE',
+})
+
+export const logout = () => {
+  util.cookieDelete('X-SailPoint-Token')
+  return tokenRemove()
+}
+
```

Comment:
signup and login could be better named `signupRequest` and `loginRequest` so the interface could distinguish between sync and async action creators